study-builder/tenants/testboston/pages/login.html

```diff
@@ -78,6 +78,20 @@
     // Decode utf8 characters properly
     var config = JSON.parse(decodeURIComponent(escape(window.atob('@@config@@'))));
     config.extraParams = config.extraParams || {};
+    var inviteKey = 'invitationId';
+
+    // save the invitation in local storage to minimize breached password loss-of-invitation
+    if (config.extraParams.invitation_id) {
+        localStorage.setItem(inviteKey,config.extraParams.invitation_id);
+        console.log('Saving invitation ' + config.extraParams.invitation_id);
```

Comment:
Maybe just write the last 4 characters to console?

Reply:
Yes...but I'm not sure why?

Reply:
Just paranoia.